### PR TITLE
Prep for release 0.12.1

### PR DESCRIPTION
**Pull request by eawoods**
This PR covers recent header and footer changes. Preparing for release to allow new CU identity information to be published. Tagging entire dev team as per our develop->master policies; I expect this to be a very short review as changes are minimal. We hope to merge and publish this week (Thursday morning?). Thanks all, sorry for the short notice.

screen shot 2018-12-19 at 3 42 32 pm

### CODE REVIEW

File: arxiv/base/templates/base/footer.html [hunk 1]
```diff
@@ -2,28 +2,34 @@
...
-      <li><a href="{{ url_for('help') }}">Help with using arXiv</a></li>
+      <li><a href="{{ url_for('help') }}">Help using arXiv</a></li>
       <li><a href="{{ url_for('faq') }}">FAQ</a>
     </ul>
   </div>
+</div>
+
+<div class="columns" style="border-top: 1px solid #979797">
+  <div class="column">
+    <p class="help">arXiv® is a registered trademark of Cornell University.</p>
```
Comment (erickpeirson) [inline arxiv/base/templates/base/footer.html]: Neat

Comment (mhl10) [reply arxiv/base/templates/base/footer.html]: Don't mean to be pedantic, but could we use the entity &#174; here instead?

Comment (mhl10) [reply arxiv/base/templates/base/footer.html]: Nevermind, maybe that's just github doing the conversion!

Comment (eawoods) [reply arxiv/base/templates/base/footer.html]: Good catch! My bad, fixed in quick PR to develop (upcoming).

**Pull request merged**